Project/MSVC2022/Library/MediaInfoLib.vcxproj

```diff
@@ -276,6 +276,7 @@
     <ClCompile Include="..\..\..\Source\MediaInfo\Text\File_Ttml.cpp" />
     <ClCompile Include="..\..\..\Source\MediaInfo\TimeCode.cpp" />
     <ClCompile Include="..\..\..\Source\MediaInfo\Video\File_Av1.cpp" />
+    <ClCompile Include="..\..\..\Source\MediaInfo\Video\File_Avs3.cpp" />

```

Comment:
Please same for all MSVC\* directory (stupid copy/pastes to do but we currently have different directory per MSVC version).

Reply:
\*.filters need to be adapted too.

Reply:
Actually the other projects (Automake etc) are also not updated, I'll do it, faster.